src/snowflake/snowpark/functions.py

```diff
@@ -10159,3 +10159,17 @@ def snowflake_cortex_summarize(text: ColumnOrLiteralStr):
     sql_func_name = "snowflake.cortex.summarize"
     text_col = _to_col_if_lit(text, sql_func_name)
     return builtin(sql_func_name)(text_col)
+
+
+def snowflake_cortex_sentiment(text: ColumnOrLiteralStr):
+    """
+    A string containing the text for which a sentiment score should be calculated.
+    Args:
+        text: A string containing the English text from which a summary should be generated.
+    Returns:
```

Comment:
Is it possible to add a simple example in the doc string?

Reply:
Sure, will add!

Reply:
+1